# Work out conversions into standard unit
Regions report  NY and Chicago both set benchmarks for testing indicators. The [White House gating criteria](https://www.whitehouse.gov/wp-content/uploads/2020/04/Guidelines-for-Opening-Up-America-Again.pdf) ask for a **robust testing program in place for at-risk healthcare workers, including emerging antibody testing**. It is up to a region to determine what that robust testing program looks like. 

Other reference articles:
* [NYT: Harvard researchers say there should be 152 tests per 100,000 people each day.](https://www.nytimes.com/interactive/2020/04/17/us/coronavirus-testing-states.html)
* [Vox: Testing indicators are a prerequisite for reopening.](https://www.vox.com/2020/5/8/21249880/coronavirus-testing-covid-reopening-economy-lockdowns-social-distancing)
* [NYT: CA lags in testing.](https://www.nytimes.com/2020/05/18/us/california-testing-for-covid-coronavirus.html)
* [Vox: CA's cases still rising despite locking down early ](https://www.vox.com/future-perfect/2020/5/16/21254748/california-coronavirus-cases-lockdown)
* [NYT: Chicago and LA have persistent virus spread](https://www.nytimes.com/2020/05/09/us/coronavirus-chicago.html)

NY and Chicago have listed their testing benchmarks per month. We want to convert all benchmarks into **the number of tests conducted per 1,000 residents per month**. Population assumptions are below.

## Testing Capacity
### NY
Indicator: 30 tests per 1,000 residents per month
<br>Assume: population of 8.4M in NYC
<br>Test rate per 1,000 per month: 30 tests per 1,000 residents per month
<br>Tests per day and per month: 8,400 tests/day or 252,000 tests/month.

### Chicago
Indicator: 4,500 tests per day and 135,000 tests per month
<br>Assume: population of 2.7M in Chicago
<br>Test rate per 1,000 per month: 50 tests per 1,000 residents per month
<br>Tests per day and per month: 4,500 tests/day or 135,000 per month

### LA
The goal is to see whether LA is hitting the daily targets for testing. LA should be testing between what NY and Chicago have set as their average daily testing goals, adjusting for LA County's population of 10 million. **LA needs to be hitting these goals daily until a vaccine is developed.** 

In [1]:
city = "NY"
nyc_pop = 8_400_000
rate_per_1k = 30 / 1_000
print(f"{city} tests per month: {round(nyc_pop * rate_per_1k):,}")
print(f"{city} tests per day: {round(nyc_pop * rate_per_1k / 30):,}")

NY tests per month: 252,000
NY tests per day: 8,400


In [2]:
city = "Chicago"
chicago_pop = 2_700_000
rate_per_1k = 50 / 1_000
print(f"{city} tests per month: {round(chicago_pop * rate_per_1k):,}")
print(f"{city} tests per day: {round(chicago_pop * rate_per_1k / 30):,}")

Chicago tests per month: 135,000
Chicago tests per day: 4,500


In [3]:
la_pop = 10_000_000
city = "LA"

nyc_rate = 30 / 1_000
chicago_rate = 50 / 1_000
la_rate = 45 / 1_000

# Let's try LA with NYC's rate and Chicago's rate
print(f"{city} tests per month (NYC rate): {round(la_pop * nyc_rate):,}")
print(f"{city} tests per day (NYC rate): {round(la_pop * nyc_rate / 30):,}")
print(f"{city} tests per month (Chicago rate): {round(la_pop * chicago_rate):,}")
print(f"{city} tests per day (Chicago rate): {round(la_pop * chicago_rate / 30):,}")
print(f"{city} tests per month (LA rate): {round(la_pop * la_rate):,}")
print(f"{city} tests per day (LA rate): {round(la_pop * la_rate / 30):,}")

LA tests per month (NYC rate): 300,000
LA tests per day (NYC rate): 10,000
LA tests per month (Chicago rate): 500,000
LA tests per day (Chicago rate): 16,667
LA tests per month (LA rate): 450,000
LA tests per day (LA rate): 15,000


## Hospital Capacity
### NY
Indicator: 30% of total hospital and ICU beds

### Chicago
Indicator: Fewer than 1,800 COVID-occupied hospital beds
<br>Assume: population of 2.7M in Chicago
<br>Fewer than 600 COVID-occupied ICU beds
<br>Fewer than 450 COVID-occupied ventilators
<br>Assume: population of 2.7M in Chicago
<br>Convert this to a bed rate per 100k residents

### LA
Indicator: 3-day average of ICU beds has not decreased over 14 days and at least 10% is available. 3-day average of ventilators has not decreased over 14 days and at least 20% is available.
<br>Assume: population of 10M in LA County, population of 4M in City of LA

## Testing benchmarks paired with share of positive cases and hospital capacity
A consistent testing of a large share of the population per month by itself is not enough. We must consistently hit the testing benchmarks along with a **share of 4-8% COVID-positive results**, and **have a consistent share of hospital capacity available.**
4% is necessary for virus to decrease its spread. 8% is CA's goal. 10% is what experts say is enough for mitigation.